src/translations/defaultMicrocopy.json

```diff
@@ -379,14 +379,14 @@
   "KeywordFilter.filterText": "Filter results by",
   "KeywordFilter.labelSelected": "\"{labelText}\"",
   "KeywordFilter.placeholder": "Enter a keyword…",
-  "LandingPage.schemaDescription": "The largest online marketplace.",
-  "LandingPage.schemaTitle": "Sell or rent listings | {marketplaceName}",
+  "LandingPage.schemaDescription": "An online marketplace.",
+  "LandingPage.schemaTitle": "{marketplaceName}",
   "LayoutWrapperAccountSettingsSideNav.contactDetailsTabTitle": "Contact details",
   "LayoutWrapperAccountSettingsSideNav.passwordTabTitle": "Password",
   "LayoutWrapperAccountSettingsSideNav.paymentMethodsTabTitle": "Payment methods",
   "LayoutWrapperAccountSettingsSideNav.paymentsTabTitle": "Payout details",
   "LimitedAccessBanner.logout": "Log out",
-  "LimitedAccessBanner.message": "You are logged in as {firstName} {lastName}. You have limited rights for actions.",
+  "LimitedAccessBanner.message": "You are logged in as {firstName} {lastName}. You have limited rights for actions. You cannot initiate transactions, take actions in them or send messages on behalf of transacting parties.",
```

Comment:
@thomasmalbaux did you try this? It starts to be a rather long text. I wonder if we should start styling this somehow.

![Screenshot 2023-08-14 at 15 09 34](https://github.com/sharetribe/web-template/assets/717315/5b6a392a-7102-4e09-b77e-a9af17fcef2c)